### DISCUSSION TITLE
Version 11.7.0

### DISCUSSION THREAD

**Discussion by metastable-void** [menhera-org/TabArray#194] [Announcements]
This release celebrates the primary maintainer [@metastable-void](https://github.com/metastable-void)'s 26 birthday (3 May). (Comments welcome at Twitter [@vericava](https://twitter.com/vericava))

## New features

- Per-container proxy settings.
- Debugging information page.

## Improvements and fixes

- Display up-to-date tab preview images in Panorama Grid.
- Improved user agent spoofing.
- Use consistent icons from Firefox source.
- Reproducible builds are now default.
- Translation updates.
- Dependencies updates.
- Bug fixes.

Download and install from: https://addons.mozilla.org/firefox/addon/container-tab-groups/

<hr /><em>This discussion was created from the release <a href='https://github.com/menhera-org/TabArray/releases/tag/v11.7.0'>Version 11.7.0</a>.</em>

**Comment by JohnLGalt** [comment]
Happy Early Birthday!
